gossip/src/crds.rs

```diff
@@ -303,6 +309,25 @@ impl Crds {
                     Err(CrdsError::InsertFailed)
                 } else if matches!(route, GossipRoute::PushMessage(_)) {
                     let entry = entry.get_mut();
+                    if entry.num_push_dups == u8::MAX {
+                        datapoint_info!(
+                            "gossip_crds_redundant_pull",
+                            (
+                                "origin",
+                                value.value.pubkey().to_string().get(..8),
+                                Option<String>
+                            ),
+                            (
+                                "signature",
+                                value.value.signature.to_string().get(..8),
+                                Option<String>
+                            )
```

Comment:
This probably would be too many metrics.
Can we just increment a counter in `self.stats` and periodically submit an aggregate metric like the rest of `self.stats`?

Reply:
ya spun up a 100 node test with this code and this was reporting a ton of data. 
And ya probably. let me think how we can measure the overall push coverage with an aggregate redundant pull metric. That would also avoid the bad compression trent mentioned: https://github.com/anza-xyz/agave/pull/139#discussion_r1518193684